From 61f95b624eb3c402b6901812512d48f5c7085870 Mon Sep 17 00:00:00 2001
From: Yang Deokgyu <secugyu@gmail.com>
Date: Thu, 5 Dec 2019 10:13:54 +0900
Subject: [PATCH 046/109] ODROID-XU4: arm/exynos: Add vendor name Hardkernel to
 its H/W information

Because the other devices we provided have vendor name in /proc/cpuinfo.

Signed-off-by: Yang Deokgyu <secugyu@gmail.com>
Change-Id: I81a553ea8f662016b702cbb5e543ba7a769e8d0f
---
 arch/arm/mach-exynos/exynos.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-exynos/exynos.c b/arch/arm/mach-exynos/exynos.c
index d34031c6c4d4..c57032c68ac4 100644
--- a/arch/arm/mach-exynos/exynos.c
+++ b/arch/arm/mach-exynos/exynos.c
@@ -220,7 +220,7 @@ DT_MACHINE_START(EXYNOS_DT, "Samsung Exynos (Flattened Device Tree)")
 MACHINE_END
 
 #define ODROID_MACHINE_START(name, compat)			\
-	DT_MACHINE_START(EXYNOS5422_ODROID_##name, "ODROID-"#name)	\
+	DT_MACHINE_START(EXYNOS5422_ODROID_##name, "Hardkernel ODROID-"#name)	\
 		.l2c_aux_val	= 0x3c400001,			\
 		.l2c_aux_mask	= 0xc20fffff,			\
 		.smp		= smp_ops(exynos_smp_ops),	\
-- 
2.25.1

